Client/src/Components/Search/index.jsx

```diff
@@ -2,16 +2,24 @@ import Autocomplete from '@mui/material/Autocomplete';
 import TextField from '@mui/material/TextField';
 import Chip from '@mui/material/Chip';
 import Box from '@mui/material/Box';
+import { useTheme } from "@mui/material";
 
 const teamMembers = [
   { title: 'John Doe'},
   { title: 'Jane Smith'},
   { title: 'Alex Johnson'},
 ];
 
-export default function Search() {
+/**
+ * @component
+ * @returns {JSX.Element}
+ */
```

Comment:
I like to include an example in the JSdoc as well, it's always nice to have one around for reference

```
/**
* @example
* // Renders a Search component
* <Search variant="outlined" />
*/

Probably not neccessary at this point since this component doesn't take any props right now, but an example always helps clear up documentation that might be unclear or difficult to explain

Reply:
Gotchya gotchya! I will keep that in mind for next time.